### ISSUE TITLE
Add allcontributors section

### ISSUE THREAD

**Issue by patorjk** [patorjk/JavaScript-Snake#96]
@all-contributors
please add @patorjk for code,doc,design,example.
please add @ultra17 for code,doc,design,bug.
please add @Rb64 for code,bug.

**Comment by patorjk** [comment]
Trying again.

@all-contributors
please add @patorjk for code,doc,design,example.
please add @ultra17 for code,doc,design,bug.
please add @Rb64 for code,bug.

**Comment by patorjk** [comment]
@all-contributors please add @Rb64 for code,bug.

**Comment by patorjk** [comment]
@all-contributors please add @legoman8304 for code,bug.

**Comment by patorjk** [comment]
@all-contributors please add @ashishsiot for doc.

**Comment by patorjk** [comment]
@all-contributors please add @dginovker for code,bug.

**Comment by patorjk** [comment]
@all-contributors please add @Megas4ever for code,design.

**Comment by patorjk** [comment]
@all-contributors please add @mamamia5x for code,bug.

**Comment by patorjk** [comment]
@all-contributors please add @Coteh for code,bug,doc.

**Comment by patorjk** [comment]
@all-contributors please add @yokesharun for code,design.

**Comment by patorjk** [comment]
@all-contributors please add @GregFrench for code.

**Comment by patorjk** [comment]
@all-contributors please add @KT360 for code,design.

**Comment by patorjk** [comment]
@all-contributors please add @Thusal06 for code,design.

**Comment by patorjk** [comment]
@all-contributors please add @akhill2606 for code.

**Comment by patorjk** [comment]
@all-contributors please add @Furtano for code,design.

**Comment by patorjk** [comment]
@all-contributors please add @Thunderforge for doc.

**Comment by Thunderforge** [comment]
@patorjk Please ask each person for their permission before doing this. I did not want to be added and I request that you remove me. Others may feel the same.

**Comment by patorjk** [comment]
@ Thunderforge removed.

**Comment by patorjk** [comment]
@Thunderforge I see that you've put in an issue with all-contributors. I thought I was doing something nice by highlighting the people who are _already_ highlighted in the [contributors](https://github.com/patorjk/JavaScript-Snake/graphs/contributors) sidebar. Your issue with them sounds like you're panicking that I wouldn't remove you. I removed you within **2 minutes** of you making your comment. I'm sorry that you're embarrassed that you contributed to this repo. I'll gladly remove people from the README if they don't want to be there. When using the bot users are notified, so it's not like you're secretly getting added. Again, I'm sorry this upset you so much, I thought this would be a nice thing to do. If it turns it others are equally unhappy maybe I'll just remove it from the README.

**Comment by Thunderforge** [comment]
@patorjk I appreciate that you removed me quickly. Nonetheless, I am disappointed that I was not asked about this before being added.

I am kind of a private person who doesn't want my name being shown more than it has to be. Also, being listed on the Readme in such a prominent way comes across as me being a major contributor, which I'm not. It also comes across as me endorsing the project. I contributed one time seven years ago and, to be honest, I have no memory of contributing. I have not kept up with the changes in the last seven years and so I am not comfortable with the idea of something that looks like an endorsement from me.

Yes, I did post a feature request on the all-contributors project: to add the ability to opt-out of any future maintainers adding me to a repo (especially without my permission). Other project maintainers may do the same thing that you did. At best, I'm playing whack-a-mole dealing with it, and that's a waste of my time. At worst, they may not be as forthcoming as you were and not remove me.
